src/gpuarray_blas_cuda_cublas.c

```diff
@@ -352,11 +368,23 @@ static int sgemm(cb_order order, cb_transpose transA, cb_transpose transB,
   GA_CUDA_EXIT_ON_ERROR(ctx, cuda_wait(B, CUDA_WAIT_READ));
   GA_CUDA_EXIT_ON_ERROR(ctx, cuda_wait(C, CUDA_WAIT_ALL));
 
-  CUBLAS_EXIT_ON_ERROR(ctx, cublasSgemm(h->h,
-                                        convT(transA), convT(transB), M, N, K,
-                                        &alpha, ((float *)A->ptr) + offA, lda,
-                                        ((float *)B->ptr) + offB, ldb, &beta,
-                                        ((float *)C->ptr) + offC, ldc));
+  if (cublasGemmEx != NULL && h->tensorCore) {
```

Comment:
I would not use tensor core with sgemm as this downcast the input to float16 before computation with float32 output.

I don't have an idea of how to add tensorcore support for float32 in the user interface. I would let this for later if this get needed. Do you have an idea of how to add this easily?

Reply:
I would just disable this part.  If there is demand for this, then we'll see.

Reply:
Normally, you don't like dead code:) But if you think this is better, I'm fine with such few dead code.

Reply:
By disable I mean delete :)